src/lib.rs

```diff
@@ -1,8 +1,192 @@
 pub mod exchange;
 
+/*
+TODO:
+write unit tests for the code in this file.
+something else is code coverage not related to the file.
+get the PR done asap. Next week is the next project.
+
+ */
```

Comment:
Just checking: the tests found in this file is not related to this comment?

Reply:
Yea, ill remove it